stacks-core/create-source-binary/build-scripts/Dockerfile.linux-glibc-arm64

```
@@ -0,0 +1,26 @@
+FROM rust:bullseye as build
+
+ARG STACKS_NODE_VERSION="No Version Info"
+ARG GIT_BRANCH='No Branch Info'
+ARG GIT_COMMIT='No Commit Info'
+ARG BUILD_DIR=/build
+ARG TARGET=aarch64-unknown-linux-gnu
+WORKDIR /src
+
+COPY . .
+
+RUN apt-get update && apt-get install -y git gcc-aarch64-linux-gnu
+
+# Run all the build steps in ramdisk in an attempt to speed things up
+RUN --mount=type=tmpfs,target=${BUILD_DIR} cp -R /src/. ${BUILD_DIR}/ \
+    && cd ${BUILD_DIR} \
+    && rustup target add ${TARGET} \
+    && CC=aarch64-linux-gnu-gcc \
+    CC_aarch64_unknown_linux_gnu=aarch64-linux-gnu-gcc \
+    CARGO_TARGET_AARCH64_UNKNOWN_LINUX_GNU_LINKER=aarch64-linux-gnu-gcc \
+    cargo build --features monitoring_prom,slog_json --release --workspace --target ${TARGET} --bin stacks-node --bin stacks-inspect --bin clarity-cli --bin blockstack-cli \
+    && mkdir -p /out \
+    && cp -R ${BUILD_DIR}/target/${TARGET}/release/. /out
+
+FROM scratch AS export-stage
+COPY --from=build /out/stacks-inspect /out/blockstack-cli /out/clarity-cli /out/stacks-node /
```

i'm wondering if we can simply this on all the Dockerfiles to something like this?
```suggestion
COPY --from=build /out/*  /
```

---

i see your point as to why this woudln't work, but i think this  is solvable. 
can you open an issue to address this later?